src/channel_state.ts

```diff
@@ -581,7 +581,17 @@ export class ChannelState<StreamChatGenerics extends ExtendableGenerics = Defaul
    *
    */
   filterErrorMessages() {
-    const filteredMessages = this.latestMessages.filter((message) => message.type !== 'error');
+    const userId = this._channel.getClient().userID;
+
+    if (!userId) return;
+
+    const filteredMessages = this.latestMessages.filter(
+      (message) =>
+        !(
+          message.type === 'error' &&
+          (message.user?.id === userId || message.user_id === userId)
```

Comment:
@arnautov-anton can we add this as a param like `onlyOwnMessages` or similar? To avoid changing the method's behavior. Btw this is a bit of a weird behavior because this means that async moderated messages will be filtered for the sender, but not for receivers. But I understand the reason for this.